src/Microsoft.Identity.Abstractions/DownstreamApi/IAuthorizationHeaderProvider.cs

```diff
@@ -49,5 +49,24 @@ public interface IAuthorizationHeaderProvider
             string scopes,
             AuthorizationHeaderProviderOptions? downstreamApiOptions = null,
             CancellationToken cancellationToken = default);
+
+        /// <summary>
+        /// Creates an authorization header for calling a protected web API on behalf of a user or the application.
+        /// </summary>
+        /// <param name="scopes">The scopes for which to request the authorization header. 
+        /// Provide a single scope if the header needs to be created on behalf of an application.</param>
+        /// <param name="options">The <see cref="AuthorizationHeaderProviderOptions"/> containing information about the API
+        /// to be called and token acquisition settings. If not provided, the header will be for a bearer token.</param>
+        /// <param name="claimsPrincipal">Inbound authentication elements. In a web API, this is usually the result of the
+        /// validation of a token. In a web app, this would be information about the signed-in user. This is not useful in
+        /// daemon applications. In Microsoft.Identity.Web you rarely need to provide this parameter as it's inferred from the
+        /// context.</param>
+        /// <param name="cancellationToken">A token to cancel the operation.</param>
+        /// <returns>A string containing the authorization header, such as "Bearer token" or "PoP token".</returns>
+        Task<string> CreateAuthorizationHeaderAsync(
```

Comment:
Isn't this a major version change?

Reply:
Yes, Abstractions will increment the major version to accommodate the breaking change.